mitmproxy/tools/console/statusbar.py

```diff
@@ -281,7 +284,8 @@ def get_status(self):
         if self.master.options.mode != "regular":
             r.append("[%s]" % self.master.options.mode)
         if self.master.options.scripts:
-            r.append("[scripts:%s]" % len(self.master.options.scripts))
+            scripts = script.get_scripts()
```

Comment:
This is nonobvious, but this change violates some invariants we have. 1) This depends on `ctx` to be populated here, which I think we only guarantee for addons. 2) this change also means we run `glob` on every redraw, which hurts performance because it requires I/O. 3) We also crucially don't want to have a direct dependence on the addon (which we have by importing it).

Long story short, the correct solution here would be for the addon to a) store the list of scripts, 2) expose a command and have that called, see
```
creplay = self.master.commands.call("replay.client.count")
```
above.